gallery/lib/pages/splash.dart

```diff
@@ -85,21 +83,10 @@ class _SplashPageState extends State<SplashPage>
         duration: Duration(
           milliseconds: splashPageAnimationDurationInMilliseconds,
         ),
-        value: 1,
         vsync: this)
       ..addListener(() {
         this.setState(() {});
       });
-    if (widget.isAnimated) {
-      _launchTimer = Timer(
```

Comment:
We can remove the `_launchTimer` from `dispose` above since it's no longer used.